R/backend-access.R

```diff
@@ -167,5 +167,22 @@ sql_escape_logical.ACCESS <- function(con, x) {
   y[is.na(x)] <- "NULL"
   y
 }
+                   
+#' @export                   
+sql_escape_date.ACCESS <-  function(con, x) {
+  # Access delimits dates using octothorpes, and uses YYYY-MM-DD
+  y <- format(x, "#%F#")
```

Comment:
Can you please spell this out as `%Y-%m-%d`

Reply:
Of course, similar change made for `sql_escape_datetime`